apps/studio/components/interfaces/Database/Publications/PublicationsTables.tsx

```diff
@@ -71,7 +72,7 @@ const PublicationsTables = ({ selectedPublication, onSelectBack }: PublicationsT
               />
             </div>
           </div>
-          {!canUpdatePublications && (
+          {isPermissionsLoaded && !canUpdatePublications && (
```

Comment:
We should probably unify both loading states and put the `!canUpdatePublications` check in `isSuccess` block.